google/ggrc-core
# GGRC-4170 Add viewLink for Document

# Issue description

The frontend checks whether `viewLink` is set for an object to render a link to its info page. Lack of it makes it hard to add `open in a new tab` or `open document` links to the Document tree view / info pane (required by Document epic).

# Steps to test the changes

1. Create a Document.
2. Fetch the Document with `GET /api/documents/42` where 42 is its id.

Expected result: the response contains `viewLink` field with `/documents/42` stored in it.
Actual result: there is no `viewLink` field.

# Solution description

`viewLink` is present for instances of every model registered in `contributed_object_views`, so adding Document there solves the issue.

Also, this PR removes an unneeded view for `BackgroundTask` and orders the list to make it more readable.

# Sanity checklist




# PR Review checklist
